### ISSUE TITLE
If the video embed's offline message is too long it gets cut off

### ISSUE THREAD

**Issue by gabek** [owncast/owncast#3312]
### Share your bug report, feature request, or comment.

If somebody puts a ton of stuff in their offline message it won't all fit in a video embed `/embed/video`.
I guess we should force it to scroll if it doesn't fit within the viewport? It's not scrollable currently.

![image](https://github.com/owncast/owncast/assets/414923/7b5c11df-de44-4243-aedd-1f7939563dac)

**Comment by Anubhav-Chauhan3367** [comment]
Hi, I am a beginner. Please assign this issue to me.

**Comment by RakeshKumar010** [comment]
Please assign issue.

**Comment by gabek** [comment]
@Anubhav-Chauhan3367 Thanks! Feel free to ask any questions or discuss what your thoughts about resolving this might be.

@RakeshKumar010 Thanks for chiming in, Anubhav commented first, but feel free to look through other Owncast issues and see if there's anything that catches your attention!

**Comment by gabek** [comment]
@Anubhav-Chauhan3367 Have any questions with this or are running into any issues?

**Comment by Anubhav-Chauhan3367** [comment]
I tried to replicate this issue by creating my own web page and following instructions from [Owncast Documentation](https://owncast.online/docs/embed/?source=admin#using-the-hls-feed). to embed a test stream to my website.
Screenshot a:

![Screenshot (312)](https://github.com/owncast/owncast/assets/63549878/09441109-1b9e-43ad-a263-6ac26785cd18)
It turns out the issue was because of **scrolling="no"** attribute in the embedding code:
<pre>
<iframe
  src="https://your.host/embed/video"
  title="Owncast"
  height="350px" width="550px"
  referrerpolicy="origin"
  scrolling="no"
  allowfullscreen>
</iframe>
</pre>
When I removed this attribute, scrolling function was working fine.
Screenshot b:

![Screenshot (313)](https://github.com/owncast/owncast/assets/63549878/e389162e-9218-4bba-bd3c-f2d584556717)

Does it addresses this issue or Am I going in wrong direction? 
If it does should i change it in documentation of [this page](https://owncast.online/docs/embed/?source=admin#using-the-hls-feed)?

**Comment by gabek** [comment]
Thank you for looking at this!

Is `auto` an option for scrolling? The scroll bar looks pretty ugly for something that's embedded, but maybe it can only show if it's required to scroll? What does removing `scrolling="no"` look like if it's shorter content?

Maybe in that documentation we can explicitly say "If you require scrolling you can enable it by..." instead of telling everyone to enable it, since most people don't need it.

**Comment by Anubhav-Chauhan3367** [comment]
We can remove **`scrolling="no"`** from documentation and if the content is small enough to fit in the specified width and height for the embed, the browser will automatically remove the scrollbar.
```
<iframe
  src="https://your.host/embed/video"
  title="Owncast"
  height="350px" width="550px"
  referrerpolicy="origin"
  allowfullscreen>
</iframe>

```
![Screenshot (314)](https://github.com/owncast/owncast/assets/63549878/dd7533e3-f738-4d5c-b915-ca571d6390f7)


And for styling, we can add instructions in the documentation.
Should I go ahead and do that?

**Comment by gabek** [comment]
That sounds like a good next step to update the documentation. Thanks so much!

**Comment by Anubhav-Chauhan3367** [comment]
I wanted to let you know that I've addressed the concerns raised in [Issue #3312](https://github.com/owncast/owncast/issues/3312) by creating a pull request.

You can review the PR here: [Link to Pull Request](https://github.com/owncast/owncast.github.io/pull/125#issue-1931335327)

I would appreciate it if you could take a look at the changes and provide your feedback.
